.github/workflows/documentation.yml

```diff
@@ -0,0 +1,38 @@
+name: "Documentation"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
```

Comment:
The naming scheme is not the same as Doctrine, this needs to be changed.

```suggestion
      - "main"
```

Reply:
I keep that in mind!